lesson2/average.py

```diff
@@ -0,0 +1,10 @@
+def avar(lstx):
```

Comment:
Function should start with verb.
```suggestion
def calc_avarage(numbers):
```